wiki/planning/move_base_flex.md

```diff
@@ -0,0 +1,221 @@
+---
+# Jekyll 'Front Matter' goes here. Most are set by default, and should NOT be
```

Comment:
Since you're adding a new page, please add it to the navigation index at `_data/navigation.yml`.

See [this commit](https://github.com/RoboticsKnowledgebase/roboticsknowledgebase.github.io/pull/139/commits/cb76a0a88c00899a8ba7f8fe25568f541c4b7f1c#diff-f8949f06b04d9d8da89c490e107e04dba06c3e06fbb9000b46192062437ac187) for reference.